cmake/AddProtobuf.cmake

```diff
@@ -37,4 +37,6 @@ dawn_export_package(
   INCLUDE_DIRS "${Protobuf_INCLUDE_DIRS}"
   LIBRARIES "protobuf::libprotobuf"
   VERSION "${Protobuf_VERSION}"
+  # https://stackoverflow.com/questions/34474091/protobuf-inline-not-in-headers
```

Comment:
In line 15 of this file, how does cmake ensures that it finds that version of protobuf? protobuf does not include a version file in cmake...

Reply:
Protobuf includes a file `protobuf-config-version.cmake` alongside their `protobuf-config.cmake` which contains a version definitions
```
set(PACKAGE_VERSION "3.4.1")
```
This is in fact the same way we propagate the Dawn version and seems to be the suggested way by CMake